src/platforms/java/common/configuration/index.mdx

```diff
@@ -297,6 +297,22 @@ SENTRY_DEBUG=true
 sentry.debug=true
 ```
 
+### Disable Sentry
+
+To disable Sentry in the debug mode, use the `enabled` option:
```

Comment:
```suggestion
To disable Sentry in debug mode, use the `enabled` option:
```

Reply:
Thank you for reviewing. Found a technical issue with that sentence and fixed it.